### ISSUE TITLE
Feedback - Advanced WordPress block layouts outdated content

### ISSUE THREAD

**Issue by ghost** [WordPress/Learn#3063]
## Type of feedback

//content

## Description

In example 2, the 'fill' button is now labeled 'grow'.

## Step-by-step reproduction instructions (optional)

1. Go to 'https://learn.wordpress.org/lesson/advanced-wordpress-block-layouts/'
2. Timestamp 7:29

## Screenshots or screen recording (optional)

<img width="905" alt="Image" src="https://github.com/user-attachments/assets/f5bf96c7-d51f-4586-972b-46d95ff0f81e" />

**Comment by benazeer-ben** [comment]
@WordPress/learn-editors

# Feedback Validation Checklist:
- If this is reporting an issue, can you confirm/reproduce the issue? Yes
- What should happen next to apply the feedback?: Outdated video should get updated in order to display recent settings from WP dashboard.

**Comment by rjekic** [comment]
# Feedback Validation Checklist:
- If this is reporting an issue, can you confirm/reproduce the issue? (Yes or No): Yes
- What should happen next to apply the feedback?: Outdated video should get updated to display recent settings from the WP dashboard. If editing and updating the video is not an option, maybe we can update the transcript with the text in parentheses, like "(The option Fill is renamed to Grow in the recent WordPress version)". Or something along this line...

@WordPress/learn-editors

**Comment by Successfulsebunya** [comment]
I agree with you @rjekic we can update the transcript to Validate the Grow "fill" error.

I think that could work simply as it will reduce the wait time on WordPress TV to edit the video.

**Comment by dhruvang21** [comment]
# Feedback Validation Checklist:
- If this is reporting an issue, can you confirm/reproduce the issue? (Yes or No): 
   - YES
- What should happen next to apply the feedback?: 
   - I agree with @rjekic’s suggested quick fix for the time being. However, the video still needs to be updated with the new content, and only after that can we consider this ticket resolved and ready to close.
